src/nvim/event/loop.c

```diff
@@ -87,6 +87,29 @@ bool loop_poll_events(Loop *loop, int64_t ms)
   return timeout_expired;
 }
 
+/// If`multiqueue` has events then process them and return.
+/// Otherwise `loop` is non-NULL, run it once and process it's fast events.
```

Comment:
```suggestion
/// If `multiqueue` has events then process them and return.
/// Otherwise `loop` is non-NULL, run it once and process its fast events.
```